### DISCUSSION TITLE
How to save OutputCollection?

### DISCUSSION THREAD

**Discussion by itzthegm** [facebookexperimental/Robyn#588] [Q&A]
Hi sorry for the beginner question, I just started to explore Robyn. I see that with the newer version that the JSON files are preferred over the RDS files. As such, in the demo I see that `robyn_response` and `robyn_allocator` takes in the OutputCollect as an argument if robyn_object (eg, the RDS file) is not passed in. 

I would like to save down the OutputCollect so I can read it back in later if my session terminated for whatever reason. InputCollect seems to be savable by `robyn_write` and later calling `robyn_inputs` with the JSON path as it does here:
https://github.com/facebookexperimental/Robyn/blob/main/demo/demo.R#L321-L325

Can the same be done with OutputCollect?

**Comment by JayLiangMKT** [comment]
saveRDS(OutputCollect,path_rds_outputcollect)
OutputCollect2<-readRDS(path_rds_outputcollect)

**Reply by JayLiangMKT** [reply]
only find the way to save out as rds since the output collect is a bucket of some objects. Ping me if json answer is provided.
